cms/djangoapps/contentstore/rest_api/v0/urls.py

```diff
@@ -45,6 +46,9 @@
     ),
 
     # Authoring API
+    re_path(
+        r'^heartbeat$', heartbeat, name='heartbeat'
```

Comment:
I recommend creating a custom view that first calls `use_studio_content_api()`, as do all our authoring API views, and then, if successful, delegates to the default heartbeat view. This will come closer to showing whether the authoring API is accessible or not -- a key objective of this work.